text_generation/causal_lm/cpp/continuous_batching/apps/accuracy_sample.cpp

```diff
@@ -66,7 +66,7 @@ int main(int argc, char* argv[]) try {
         .max_num_batched_tokens = 32,
         // cache params
         .num_kv_blocks = 364,
-        .block_size = 16,
+        .block_size = 32,
```

Comment:
should this value be platform dependent? E.g. on client we could probably use smaller block_size

Reply:
Currently only 32 is supported. Ideally, the bigger value may make more continuous block and better performance.